GridMvc.Site/Models/Grids/OrdersAjaxPagingGrid.cs

```diff
@@ -26,6 +26,8 @@ public AjaxGridPager(IGrid grid)
 
         public int CurrentPage { get; set; }
 
+        public int CustomItemsCount { get; set; }
```

Comment:
This is an optional parameter, it should be nullable (`int?`).
With a normal `int` you cannot distinguish between no custom value and no items found.